WordPress/UITests/WPUITestCredentials.swift

```diff
@@ -6,7 +6,8 @@ struct WPUITestCredentials {
     static let testWPcomUsername: String = "e2eflowtestingmobile"
     static let testWPcomPassword: String = "pw"
     static let testWPcomSiteAddress: String = "tricountyrealestate.wordpress.com"
-    static let testWPcomSitePrimaryAddress: String = "tricountyrealestate.wordpress.com"
+    static let testWPcomPrimaryPaidSite: String = "tricountyrealestate.wordpress.com"
+    static let testWPcomSecondaryFreeSite: String = "fourpawsdoggrooming.wordpress.com"
```

Comment:
A couple of things here:

- `testWPcomSiteAddress` and `testWPcomPrimaryPaidSite` have the same value, have you considered keeping only one of them?
- It could be just me, but `testWPcomPrimaryPaidSite` and `testWPcomSecondaryFreeSite` gives me the impression that there could be a SecondaryPaidSite or a PrimaryFreeSite. As the site type says more about it than Primary or Secondary, WDYT about making them `testWPcomPaidSite` and `testWPcomFreeSite`?

Reply:
thanks for pointing that out! that's an existing value but definitely should use this as an opportunity to clean that up. 

for the second point, i can see what you mean, i reused "primary" from the original name but yeah `testWPcomPaidSite` and `testWPcomFreeSite` is more concise, i'll update to the suggestion - updated in 70a655f214ecf4b48c611485382f64d675b528ed